omnibus/config/software/server-complete.rb

```diff
@@ -0,0 +1,67 @@
+name "server-complete"
+
+skip_transitive_dependency_licensing true
+
+license :project_license
+
+# global
+dependency "libiconv"
```

Comment:
This makes me a bit nervous as I feel like this means we have under-specified software definitions here.  Perhaps a comment about why this is at the top level?